uber/models.py

```diff
@@ -296,6 +296,9 @@ def _ints(self, name, val):
 
     @suffix_property
     def _label(self, name, val):
+        if not val:
+            return ''
```

Comment:
This is actually important to export/importing - it turns out that if you export labels that don't have corresponding integers, the import will fail later.